packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,50 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
+import getSingletonElementInstance from "./getSingletonElementInstance.js";
+
+const styles = `position: absolute;
+	clip: rect(1px,1px,1px,1px);
+	user-select: none;
+	left: -1000px;
+	top: -1000px;
+	pointer-events: none;`;
+
+const politeSpan = document.createElement("span");
+const assertiveSpan = document.createElement("span");
+
+politeSpan.classList.add("ui5-invisiblemessage-polite");
+assertiveSpan.classList.add("ui5-invisiblemessage-assertive");
+
+politeSpan.setAttribute("aria-live", "polite");
+assertiveSpan.setAttribute("aria-live", "assertive");
+
+politeSpan.setAttribute("role", "alert");
+assertiveSpan.setAttribute("role", "alert");
+
+politeSpan.style.cssText = styles;
+assertiveSpan.style.cssText = styles;
+
+if (!politeSpan.parentElement) {
```

Comment:
Wouldn't this be always true?

Reply:
Done.

Reply:
Please check if you have pushed the changes.